src/main/kotlin/com/zopa/ktor/opentracing/OpenTracingServer.kt

```diff
@@ -66,6 +72,15 @@ class OpenTracingServer(
                 if (clientSpanContext != null) spanBuilder.asChildOf(clientSpanContext)
 
                 val span = spanBuilder.start()
+
+                config.tagsToAdd.forEach {
+                    try {
+                        span.setTag(it.first, it.second.invoke())
+                    } catch (e: Exception)  {
+                        log.warn(e) { "Could not add tag: ${it.first}" }
+                    }
+                }
```

Comment:
this also needs to be done in the client spans here 
https://github.com/zopaUK/ktor-opentracing/blob/67f025cbaaf6af100ac4853b6b4af63003e36aa4/src/main/kotlin/com/zopa/ktor/opentracing/OpenTracingClient.kt#L40